openpaas-james/apps/memory/src/test/java/com/linagora/openpaas/james/app/EncryptedMemoryServerTest.java

```diff
@@ -84,6 +84,8 @@ public void shouldUseEncryptedMailboxManager(GuiceJamesServer jamesServer) {
     }
 
     @Test
+    @Disabled("ISSUE-96: Unstable. Creates a 404 error for `connectJMAPServerShouldRespondBadRequest` test in " +
+        "MemoryServerTest if that test suite if played after EncryptedMemoryServerTest, for some unknown reason.")
```

Comment:
I could not find any better TBH. I can reproduce that failed build on my local with maven. Maven (at least for me) seems playing tests classes in alphabetical order. So it plays `EncryptedMemoryServerTest`, then `MemoryServerTest`.

When played in that order, the jmap test for `MemoryServerTest` fails (404). If I remove that test from `EncryptedMemoryServerTest`, then `MemoryServerTest` is green.

Don't ask me what is wrong with this test and why it potentially affects the following test class, I didn't get it. This test itself is green, but makes the jmap test in an other class failing. No idea what is causing this...

Reply:
If somebody finds the solution or something better, I'm listening :)

Reply:
I did run `mvn test` locally...

Reply:
CI is the source of truth...